homework-projects/kaustav_bakshi/file-1.adoc

```diff
@@ -0,0 +1,186 @@
+[[file-1]]
+= file 1
+
+== *3a. Describe the product: benefits, what it does.* +
+**About Peek +
+**Peek is useful for creating tutorials, bug reports, demonstrations and screen +
+capture in general. It is easy to use and provides a simple and efficient way to +
+share information with others. +
+ +
+**What it does +
+**Users can select a specific region of their screen to record, adjust the frame rate +
+and quality of the recording, and add annotations such as text or arrows. Once +
+the recording is complete, the user can preview it before saving it as a GIF or +
+video file. The recording can also be directly uploaded to various platforms like +
+Imgur, Dropbox, etc. +
+ +
+*Benefits of Peek* +
+● After recording, Peek allows users to easily share their recordings with +
```

Comment:
Should be replaced https://docs.asciidoctor.org/asciidoc/latest/lists/unordered/